pkg/template/helm_template_test.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package template
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path"
+	"reflect"
+	"testing"
+
+	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/memdir"
+)
+
+func TestHelmTemplateCmdLookup(t *testing.T) {
+	tests := []struct {
+		chartSpecVersion string                // i.e v1 or v2
+		binaryName       string                // returned helm binary name
+		args             func(string) []string // returned helm template arguments
+	}{
+		{
+			"v1",
+			helm2BinaryName,
```

Comment:
ideally tests should not use same constants that they are testing. let's hard code these values.

Reply:
Makes sense, I had mixed feelings when i was writing it. Thanks for pointing it out! :)